secant/secant-mainnet-Info.plist

```diff
@@ -23,17 +23,6 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
-	<key>CFBundleURLTypes</key>
-	<array>
-		<dict>
-			<key>CFBundleTypeRole</key>
-			<string>Editor</string>
-			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>zcash</string>
```

Comment:
Looking at https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app#Register-your-URL-scheme and https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleurltypes, this change seems correct to me. I also found this lovely note:
> Note: If multiple apps register the same scheme, the app the system targets is undefined. There’s no mechanism to change the app or to change the order apps appear in a Share sheet.
which gives even further impetus to not use them (Android does allow choosing this for custom URL schemes).

I note that this is not removed from the testnet app plist. I initially presumed that might be because it is still useful there, but AFAICT we don't even have the [app delegate handling code](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app#Handle-incoming-URLs) for these URIs:
https://github.com/Electric-Coin-Company/zashi-ios/blob/6970b6ff60f574dd8277bf398ef595819ac0fa0f/secant/AppDelegate.swift#L31-L55